### ISSUE TITLE
Padding for titlebar when app is running in full screen mode

### ISSUE THREAD

**Issue by anthonywebb** [touchstonejs/touchstonejs#22]
In iOS 8 (and likely previous versions) you may place a shortcut to the web app on the home screen.  When you do this and you launch from the home screen the webpage will show up without any browser chrome (which is awesome BTW)

Problem is that there is no padding at the top of the webview to accomodate for the device statusbar which means the webpage bleeds into it.

Is anyone aware of a way to detect that the app is running full screen so the appropriate padding can be added?

![img_0600](https://cloud.githubusercontent.com/assets/4397/5658943/4e452472-96c7-11e4-9d64-f86f64f29e49.PNG)

**Comment by anthonywebb** [comment]
I found a link detailing stuff.  It appears as though you are able to tell if the app has been launched in full screen or not.  Looks like:
if (window.navigator.standalone) {
    // The app is running in standalone mode so you may need 20 pixels of padding
}

Looks like you only need the padding if you have a meta tag of "apple-mobile-web-app-status-bar-style" set to "black-translucent" (like the touchstone demo app is).  I just changed mine to "black" and it doesnt bleed into the titlebar like before (alas I have a black titlebar now)

Details here: http://blog.teamtreehouse.com/optimizing-mobile-web-apps-ios
